nemo/collections/nlp/models/language_modeling/megatron/griffin/griffin_block.py

```diff
@@ -13,10 +13,13 @@
 # limitations under the License.
 
 from megatron.core.models.common.language_module.language_module import LanguageModule
-from megatron.core.transformer.custom_layers.transformer_engine import TENorm
+from megatron.core.transformer.custom_layers.transformer_engine import TENorm, te_checkpoint
```

Comment:
Add guard to Mcore import https://github.com/NVIDIA/NeMo/blob/8c1ce65961c60df8c58817cae6f1cb7b5e5d407a/nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py#L88